src/components/frame/Extensions/layouts/PageLayout.tsx

```diff
@@ -87,11 +96,11 @@ const ExtensionsLayout: FC<PropsWithChildren> = ({ children }) => {
       </div>
       <main className="mt-9 pb-24">
         <div className="inner">
-          {isMobile && <Navigation />}
+          {isMobile && <Navigation pageName={pageName} />}
           <div className="mt-9 sm:mt-6">
             <PageTitle
-              title={formatMessage({ id: 'extensionsPage.title' })}
-              subtitle={formatMessage({ id: 'extensionsPage.description' })}
+              title={formatMessage(title)}
```

Comment:
Maybe would be better to move formatMessage to the PageTitle component?

Reply:
moved to pageTitle